docs/review/fw-adr-0031-opencode-hook-bridge-review.md

```
@@ -0,0 +1,1152 @@
+# Code Review — OpenCode enforcement hook bridge (fw-adr-0031)
+
+<!-- TOC -->
+
+- [1. Identification](#1-identification)
+- [2. Scope and method](#2-scope-and-method)
+- [3. Original review record — commit 6244214](#3-original-review-record-commit-6244214)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)